packages/mongo-nav/src/mongo-select/ProjectSelect.tsx

```diff
@@ -234,102 +231,109 @@ function ProjectSelect({
-        className={menuContainerStyle}
-        justify="start"
-        spacing={0}
-        open={open}
-        setOpen={toggleOpen}
-        data-testid="project-select-project-list"
-      >
-        <FocusableMenuItem>
-          <Input
-            data-testid="project-filter-input"
-            onChange={onChange}
-            onKeyDown={onKeyDown}
-            variant="project"
-            value={value}
-          />
-        </FocusableMenuItem>
-
-        <ul className={ulStyle}>
-          {isAdminSearch && isFetching && (
-            <li className={emptyStateStyle}>Searching...</li>
+      <div>
+        <button
+          {...rest}
+          {...projectTriggerDataProp.prop}
+          ref={setButtonElement}
+          onClick={onElementClick(
+            NavElement.ProjectNavProjectSelectTrigger,
+            toggleOpen,
           )}
-          {isAdminSearch && !isFetching && renderedData.length === 0 && (
-            <li className={emptyStateStyle}>No matches found</li>
+          data-testid="project-select-trigger"
+          className={cx(
+            baseButtonStyle,
+            projectTriggerStyle,
+            {
+              [activeButtonStyle]: open,
+              [textLoadingStyle]: loading,
+            },
+            className,
           )}
-          {renderedData?.map(renderProjectOption)}
-        </ul>
-
-        <MenuSeparator />
-
-        <li onKeyDown={onKeyDown} role="none" className={projectButtonStyle}>
-          <FocusableMenuItem>
-            <Button
-              href={urls.viewAllProjects as string}
-              data-testid="project-select-view-all-projects"
-              onClick={onElementClick(NavElement.ProjectNavViewAllProjects)}
-            >
-              View All Projects
-            </Button>
-          </FocusableMenuItem>
+          disabled={loading}
+          aria-disabled={loading}
+        >
+          <FolderIcon
+            className={cx(iconColorStyle, { [iconLoadingStyle]: loading })}
+          />
+          <span
+            className={cx(selectedStyle, { [textLoadingStyle]: loading })}
+            data-testid="project-select-active-project"
+          >
+            {current?.projectName ?? ''}
+          </span>
+
+          <CurrentIcon
+            size="small"
+            className={cx(caretBaseStyle, { [iconLoadingStyle]: loading })}
+          />
+        </button>
+        <Menu
+          usePortal={false}
+          className={menuContainerStyle}
+          justify="start"
+          spacing={0}
+          open={open}
+          setOpen={toggleOpen}
+          data-testid="project-select-project-list"
+        >
           <FocusableMenuItem>
-            <Button
-              href={urls.newProject as string}
-              data-testid="project-select-add-new-project"
-              onClick={onElementClick(NavElement.ProjectNavAddProject)}
-            >
-              + New Project
-            </Button>
+            <Input
```

Comment:
ditto here: 
<img width="350" alt="Screen Shot 2020-11-17 at 11 26 53 AM" src="https://user-images.githubusercontent.com/26016393/99417590-153fb900-28c8-11eb-86b3-474bede27cec.png">

Reply:
this also seems to show a bug with rendering Button as an anchor link